config/initializers/revision.rb

```diff
@@ -0,0 +1,5 @@
+revision_file = Rails.root.join 'public/commit.txt'
```

Comment:
Hmm, where is that file coming from? The dockerfile puts it in `/app/commit_id.txt` I think.

https://github.com/zooniverse/education-api/blob/master/Dockerfile#L19